kedro-datasets/pyproject.toml

```diff
@@ -17,8 +17,8 @@ dependencies = [
 dynamic = ["readme", "version"]
 
 [project.optional-dependencies]
-pandas-base = ["pandas>=1.3, <3.0",]
-spark-base = ["pyspark>=2.2, <4.0",]
+pandas-base = ["pandas>=2.0, <3.0",]
+spark-base = ["pyspark>=3.0, <4.0",]
```

Comment:
This is a breaking change and needs a changelog entry

Reply:
For full completeness, there was some pushback against dropping support for pandas 1.x given that it still has many downloads. Reverted these in https://github.com/kedro-org/kedro-plugins/pull/646 without actually changing the test requirements to see if tests pass